vochain/state/process.go

```diff
@@ -161,6 +162,92 @@ func (v *State) CountProcesses(committed bool) (uint64, error) {
 	return count, nil
 }
 
+// RegisterStartBlock function creates a new record on the key-value database
+// associated to the processes subtree for the process ID and the startBlock
+// number provided.
+func (v *State) RegisterStartBlock(pid []byte, startBlock uint32) error {
```

Comment:
I think the name of the method is not self-descriptive. I'd be more explicit: `ProcessRegisterStartBlockNoState()`

Reply:
Actually I think we might make a better structure to clearly differentiate this methods, since its hard to understand what is the actual purpose.

To this end I'd propose to create a new type: `ProcessBlockRegistry`

On the `State` constructor, we can create a new instance for this type:

`State.ProcessBlockRegistry = ProcessBlockRegistry{ treeRW: processTree.NoState(), treeRead: ....,  treeLock: v.Tx }`

Then whenever you want to use it, just:

`state.ProcessBlockRegistry.SetStartBlock(pid, startBlock)`

So all methods are contained inside the ProcessBlockRegistry.